tdtax/eruptive.yaml

```
@@ -0,0 +1,51 @@
+class: Eruptive
+subclasses:
+  - class: UV Ceti
+    tags: [types M/K, dwarf, magnetic reconnection]
+    other names: [UV, Flare star, UV Cet-type]
+  - class: S Doradus
+    tags: [type B, supergiant/hypergiant, S Doradus instability strip, high luminosity]
+    other names: [S Dor, SDor, SDOR, Luminous blue variable, LBV]
+  - class: Wolf-Rayet
+    tags: [broad lines]
+    other names: [WR]
+  - class: Gamma Cas
+    tags: [Be star, accretion disk]
+    other names: [Gamma Cassiopeiae, GCas, GCAS]
+    subclasses:
+      - class: Be
+        other names: []
+      - class: Shell
+        other names: []
+  - class: FU Ori
+    tags: [pre-main sequence, accretion disk, T Tauri evolution, possible Orion subtype]
+    other names: [FU, FUor, FU Ori, FU Orionis]
+  - class: Orion
+    tags: [pre-main sequence, diffuse nebulae]
+    other names: [IN]
+    subclasses:
+      - class: w/Abs
+        other names: [IN(YY)]
+      - class: T Tauri
+        tags: [types F/G/K/M, pre-main sequence, strong chromospheric lines, Hayashi track]
+        other names: [TTS, TT, INT, IT]
+        subclasses: 
+          - class: TTc
+            other names: [Classical TTS, Classical T Tauri]
+          - class: TTw
+            tags: [Weak emission lines]
+            other names: [Weak-line TTS, Weak-line T Tauri]
+  - class: R Cor Borealis
+    tags: [types F/G, supergiant]
+    other names: [R Coronae Borealis, RCB, R CrB, R Cor Bor]
+    subclasses:
+       - class: DY Per
+         other names: [DY Persei]
+         tags: [AGB, carbon-rich]
+         links: [https://en.wikipedia.org/wiki/DY_Persei_variable]
```

add quotes to make this a valid YAML:
```suggestion
         links: ["https://en.wikipedia.org/wiki/DY_Persei_variable"]
```